src/ccutil/unicharset.cpp

```diff
@@ -706,6 +709,7 @@ bool UNICHARSET::save_to_string(STRING *str) const {
               this->get_script_from_script_id(this->get_script(id)),
               this->get_other_case(id));
     } else {
+      // FIXME
```

Comment:
You should clarify what is needed to be fixed here

Reply:
... by adding a comment after the FIXME

Reply:
I should have done that, yes. But instead of fixing the FIXME comment, I prefer to fix the code, hopefully today.

Reply:
OK :-)

Reply:
`save_to_string` is fixed now, too.